### DISCUSSION TITLE
Third-Party Audit/Accreditation

### DISCUSSION THREAD

**Discussion by ZackSilva** [Security-Onion-Solutions/securityonion#13852] [Unsupported Versions]
I do not see evidence of a third-party audit having been conducted against Security Onion. Does it have any past audits or standing accreditations in any capacity?

**Comment by dougburks** [comment]
Yes, Security Onion has gone through third-party audits and has been accredited for use in many organizations and environments.

Our source code is available on this Github repo so please feel free to perform your own audit. 

You may also be interested in this section of our documentation:
https://docs.securityonion.net/en/2.4/security.html#product-and-supply-chain-integrity
